src/main/java/edu/usf/cutr/gtfsrtvalidator/helper/TimeStampHelper.java

```diff
@@ -18,9 +18,10 @@
 package edu.usf.cutr.gtfsrtvalidator.helper;
 
 import java.util.Date;
+import java.util.concurrent.TimeUnit;
 
 public class TimeStampHelper {
     public static long getCurrentTimestamp() {
-        return (long) Math.ceil((double) new Date().getTime() / 1000);
+        return TimeUnit.MILLISECONDS.toSeconds(new Date().getTime());
```

Comment:
Actually, looks like we might need to address the precision issue in this PR after all.  I'll comment shortly in my review.